application/src/test/resources/logback-test.xml

```diff
@@ -21,6 +21,13 @@
     <!-- mute TelemetryEdgeSqlTest that causes a lot of randomly generated errors -->
     <logger name="org.thingsboard.server.service.edge.rpc.EdgeGrpcSession" level="OFF"/>
 
+    <!--    LwM2m lifecycle debug for the test scope -->
+    <logger name="org.thingsboard.server.transport.lwm2m.server.downlink.DefaultLwM2mDownlinkMsgHandler" level="TRACE"/>
```

Comment:
Please comment this lines since it may cause the flood of trace messages, right?

Reply:
It is temporary for tests only. Only a little output is produced during tests. Just in case any fluky behavior happens in the nearest future.

Reply:
![image](https://user-images.githubusercontent.com/79898499/224333385-00a3a74f-a637-439a-96d1-e86236fcae6e.png)